test.sh

```diff
@@ -110,7 +110,7 @@ vagrantDestroy
 nodesOff
 
 # Power on vagrant box and nodes 
-vagrantUp
+vagrantUp "RackHD/packer/ansible/roles/monorail/files/auth_config"
```

Comment:
@jlongever  Is there misunderstanding here ?
The auth config has **both** endpoints with auth **enabled** and **disabled**. So the smoke test will cover both auth and no auth cases. won't skip any original tests with no auth.
```JSON
    "httpEndpoints": [
        {
            "address": "0.0.0.0",
            "port": 8080,
            "httpsEnabled": false,
            "proxiesEnabled": true,
            "authEnabled": false,
            "httpsCert": "data/dev-cert.pem",
            "httpsKey": "data/dev-key.pem",
            "routers": "northbound-api-router"
        },
        {
            "address": "0.0.0.0",
            "port": 8443,
            "httpsEnabled": true,
            "proxiesEnabled": true,
            "authEnabled": true,
            "routers": "northbound-api-router"
        },
        {
            "address": "172.31.128.1",
            "port": 9080,
            "httpsEnabled": false,
            "proxiesEnabled": true,
            "authEnabled": false,
            "routers": "southbound-api-router"
        }
    ]
```